modules/openapi-generator/src/test/java/org/openapitools/codegen/postman/PostmanCollectionCodegenTest.java

```diff
@@ -602,11 +602,12 @@ public void convertObjectNodeIncludingDoubleQuoteToJson() {
     @Test
     public void convertLinkedHashMapToJson() {
 
-        final String EXPECTED = "{\\n \\\"id\\\": 1,\\n \\\"city\\\": \\\"Amsterdam\\\"\\n}";
+        final String EXPECTED = "{\\n \\\"id\\\": 1,\\n \\\"city\\\": \\\"Amsterdam\\\",\\n \\\"safe\\\": true\\n}";
 
         LinkedHashMap<String, Object> city = new LinkedHashMap<>();
         city.put("id", 1);
         city.put("city", "Amsterdam");
+        city.put("safe", true);
```

Comment:
in addition to this test, what about adding another test to `modules/openapi-generator/src/test/resources/3_0/postman-collection/SampleProject.yaml` as well?

Reply:
feel free to add the test later